src/pages/menuUpload/CustomerTypePage.tsx

```diff
@@ -105,18 +146,47 @@ function CustomerTypePage() {
             onCancel={toggleEditModal}
             title="Edit Customer Type"
             initialFormValue={objectToEdit}
+            submitButtonText="Save"
+            mode="Edit"
+            onDelete={onDeleteSubmitEdit}
+          />,
+        ]}
+      />
+      <Dialog
+        isOpen={displayDeleteModal}
+        onClose={toggleDeleteModal}
+        setCustomDialogContent
+        // eslint-disable-next-line
+        children={[
+          <CustomerTypeForm
+            onSubmit={onSubmitDelete}
+            onCancel={toggleDeleteModal}
+            title="Delete Customer Type"
+            initialFormValue={objectToDelete}
+            submitButtonText="Delete"
+            mode="Delete"
           />,
         ]}
       />
       {customerTypes === undefined ? (
-        <Table columns={columns} data={[{ }]} editAction={toggleEditModal}>
+        <Table
+          columns={columns}
+          data={[{ }]}
+          editAction={toggleEditModal}
```

Comment:
Here Table is empty, right? So `editAction` and `deleteAction` props I think are unnecessary ;)

Reply:
ok.